### PR TITLE
Seeding

### PR DESCRIPTION
**Pull request by Gosuguy**
Created seeding script to be able to import DB to Postgres and MongoDB

### CODE REVIEW

File: utilities/_dbSeeder.js [hunk 1]
```diff
@@ -24,7 +26,40 @@ const _dbSeeder = () => {
...
+  // let fakeInfoArr = [];
+  let counter = 0;
+  while (counter < 500000) {
+    // fakeInfoArr.push([
+      addToDatabase([
+      faker.fake('{{commerce.productName}}'),
+      faker.fake('{{lorem.lines}}'),
+      faker.fake('{{commerce.price}}'),
+      faker.fake('{{internet.userName}}'),
+      (() => {
```
Comment (auramix) [inline utilities/_dbSeeder.js]: Consider breaking this off into a helper function for readability

File: utilities/createCSV.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+const writeCsv = async () => {
+  for (var i = 0; i < 10000000; i++) {
+    await csvWriter
+    .writeRecords([{
+    // id: i + 1,
+    title: faker.fake('{{commerce.productName}}'),
+    description: faker.fake('{{lorem.lines}}'),
+    product_price: faker.fake('{{commerce.price}}'),
+    seller: faker.fake('{{internet.userName}}'),
+    colors: '{' + (() => {
```
Comment (auramix) [inline utilities/createCSV.js]: Consider breaking this off into a helper function for readability

**Review by auramix — COMMENTED** [review]
Looks good, Juan. Made two suggestions for helper functions to aid readability.

File: database/index.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-const mongoose = require('mongoose');
+/*const mongoose = require('mongoose');
```
Comment (ruparocks) [inline database/index.js:1]: Your seeding script does not touch your DB. You should uncomment this code so your app works, or you should remove the code if it is truly unneeded.

File: database/index.js [hunk 4]
```diff
@@ -79,9 +79,76 @@ const deleteFromDatabase = (id) => {
   ProductInfo.remove({id}, function(err) { // TO RESET COLLECTION
     console.log('collection removed')
   });
-}
+};
 
 module.exports.updateDatabase = updateDatabase;
```
Comment (ruparocks) [inline database/index.js:84]: Code convention is to put module.exports at the bottom of your file. 
I recommend updating this to ES6 syntax: `module.exports = {updateDatabase, queryDatabase, queryAllFromDatabase, deleteFromDatabase};` or `export { func1, func2, func3 };`.

File: database/index.js [hunk 5]
```diff
@@ -79,9 +79,76 @@ const deleteFromDatabase = (id) => {
...
 module.exports.queryDatabase = queryDatabase;
 module.exports.queryAllFromDatabase = queryAllFromDatabase;
 module.exports.deleteFromDatabase = deleteFromDatabase;
+*/
+
+/////////////////////////////////////////////////////////
```
Comment (ruparocks) [inline database/index.js:90]: Clean out comments that are not adding value

File: package.json [hunk 6]
```diff
@@ -5,8 +5,9 @@
     "start": "concurrently --kill-others \"npm run server\" \"npm run react\" ",
     "build": "webpack -p",
     "test": "jest tests/",
-    "seedDb": "node utilities/_dbSeeder.js",
-    "database": "mongod"
+    "seedDb": "concurrently \"node utilities/_dbSeeder.js\" \"node utilities/_dbSeeder.js\" \"node utilities/_dbSeeder.js\" \"node utilities/_dbSeeder.js\" \"node utilities/_dbSeeder.js\"",
```
Comment (ruparocks) [inline package.json:8]: Are you still using this command? If you are not, please remove it.

File: utilities/createCSV.js [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+      return resultArr;
+    })() + '}',
+    }], (err, dataToWrite) => {
+    console.log('Added',dataToWrite, 'to CSV')
+    });  
+    if (i % 10000 === 0) console.log('Reached ', i, '!');
+  }
+};
+writeCsv();    
+/*
```
Comment (ruparocks) [inline utilities/createCSV.js:43]: Again, remove all commented code from your codebase. This adds technical debt. I do not accept any PRs with commented out code. Comments should be used for debugging statements and comments that explain complex logic.

File: utilities/seed.csv [hunk 8]
```diff
@@ -0,0 +1,33 @@
+title,description,product_price,seller,colors
```
Comment (ruparocks) [inline utilities/seed.csv:1]: Does this file need to be in source control?

**Review by ruparocks — COMMENTED** [review]
There are too many commented lines of code on the PR and files that should be ignored. 
.DS_Store is not needed in the PR. Do you need the CSV files? It should be clear what npm script will generate the data.

**Comment by ruparocks** [comment]
This PR is not ready to be merged, there are comments and suggestions that need to get addressed. Juan you should refactor your code.

**Pull request merged**